apple/LayoutReanimation/REASnapshot.m

```diff
@@ -110,26 +146,31 @@ - (void)makeSnapshotForView:(REAUIView *)view useAbsolutePositionOnly:(BOOL)useA
   }
 }
 
-- (REAUIView *)findTransformedView:(REAUIView *)view
+- (CGAffineTransform)findCombinedTransform:(REAUIView *)view
 {
-  REAUIView *transformedView;
-  bool isTransformed = false;
-  do {
-    if (transformedView == nil) {
-      transformedView = view;
-    } else {
-      transformedView = transformedView.superview;
+  CGAffineTransform transform = view.transform;
+  view = view.superview;
+  while (view != nil && ![REAScreensHelper isRNSScreenType:view]) {
+    // ignore transforms that are caused by transitions
+    if (![view.superview isKindOfClass:NSClassFromString(@"UITransitionView")]) {
+      CGAffineTransform t = view.transform;
+      // we are ignoring translations in superviews, since the positioning obtained by converting the center point (and
+      // applying it's view transforms) to the main window is already correct, here we only care about scale, shear and
+      // rotation
+      transform = CGAffineTransformConcat(transform, CGAffineTransformMake(t.a, t.b, t.c, t.d, 0, 0));
     }
-    CGAffineTransform transform = transformedView.transform;
-    isTransformed = transform.a != 1 || transform.b != 0 || transform.c != 0 || transform.d != 1 || transform.tx != 0 ||
-        transform.ty != 0;
-  } while (!isTransformed &&
-           transformedView != nil
-           // Ignore views above screen
-           && ![REAScreensHelper isRNSScreenType:transformedView]);
+    view = view.superview;
+  }
+  return transform;
+}
 
-  if (isTransformed && transformedView != nil) {
-    return transformedView;
+- (REAUIView *)maybeFindTransitionView:(REAUIView *)view
+{
+  while (view != nil && ![REAScreensHelper isRNSScreenType:view]) {
+    if ([view.superview isKindOfClass:NSClassFromString(@"UITransitionView")]) {
```

Comment:
I'm curious to know what's the difference between this and 
```suggestion
    if ([view.superview isKindOfClass:[UITransitionView class]]) {
```

Reply:
Unfortunately we can't because this is an internal Apple API and not published in the SDK :/